### ISSUE TITLE
[Keyboard Navigation - Thresholds]: “Delete” Buttons are not accessible through Keyboard under “Thresholds “section.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66254]
### Repro Steps: 
Open #Url: https://play.grafana.org/ Sign in with valid credentials. "Home" page appears. Navigate the Feature Showcase pane heading and select it. Navigate the expanded Dropdown and select the "Edit" menu item. Navigate the page "Edit" page which appears. Navigate to the "All" buttons and select it. Navigate to the “Thresholds" button and expand it. Navigate all the controls present under it. Run AI Tool. Verify the issue.

### Actual Result: Issue: 
“Delete” Buttons are not accessible through Keyboard under “Thresholds “section. Expected Result:

### Solution: 
“Delete” Buttons should be accessiblethrough keyboard under “Thresholds “section.
Note: The same issue exists "List items' present under "Visualizations tab". (Home page>Featured showcases>Edit>Change to visualization button>Visualizations tab) Same issue repro for the "Clear tags" control present under "Search dashboards" screen.

### Severity
`SEV2`
